cmd/systray/command/command.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/DataDog/datadog-agent/comp/core/config"
 	"github.com/DataDog/datadog-agent/comp/core/flare"
 	"github.com/DataDog/datadog-agent/comp/core/log/logimpl"
+	rcsetting "github.com/DataDog/datadog-agent/comp/core/settings"
```

Comment:
This aliased import looks misleading. These are not RC settings. Can the alias be renamed?